pulp_container/app/viewsets.py

```diff
@@ -594,6 +621,30 @@ class ContainerPushRepositoryVersionViewSet(RepositoryVersionViewSet):
     """
 
     parent_viewset = ContainerPushRepositoryViewSet
+    permission_classes = (access_policy.RepositoryVersionAccessPolicyFromDB,)
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": ["retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition":
+                    "has_repoversion_repo_model_or_obj_perms:container.view_containerpushrepository"
+            },
+            {
+                "action": ["destroy"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_pushrepoversion_delete_perms",
```

Comment:
I made this one non-customizable. 
If I got perms right and if we want to make this one more customizable, I can introduce lookups by distribution and by namespace and we'll have a looooong condition with ORs here.

Any opinions?

Reply:
Alternatively, we can add a delete_containerpushrepo to the auto-assignment section of the push repo viewset and check it here.
I came up with this only now, while writing the comments.

Reply:
adding 'delete_repo_versions' in the auto-assignment section of the container push repo would make sense to me.

Reply:
after thinking a bit more..i am not sure it is a good idea to allow to delete repo versions from push repo.
example: you perform podman push. At the end of the operation  you have 3 repo versions, because that image had 2 blobs and 1 manifest. If the user will remove last repo version it will make the repo unusable. Since the user does not know these subtle details of 'complete' incomplete versions i suggest we do not allow deleting versions at all.
Unless we can enable 'delete any version expect the last one" ( last version is always served of the container push repo)

Reply:
It make sense to me, I'll remove destroy section from here.